test/html.js

```diff
@@ -198,11 +198,11 @@ describe('HTML Parser', function () {
 
 		describe('#removeWhitespace()', function () {
 			it('should remove whitespaces while preserving nodes with content', function () {
-				const root = parseHTML('<p> \r \n  \t <h5>  123  </h5></p>');
+				const root = parseHTML('<p> \r \n  \t <h5>  123&nbsp;  </h5></p>');
 
-				const textNode = new TextNode('  123  ');
-				textNode.rawText = textNode.trimmedText;
-				textNode.rawText.should.eql(' 123 ');
+				const textNode = new TextNode('  123&nbsp;  ');
+				textNode.rawText = textNode.trimmedRawText;
+				textNode.rawText.should.eql(' 123&nbsp; ');
```

Comment:
I wrote this original test in a previous PR. We're altering it slightly to make sure we're using the proper rawText version of trimmed text. We also add in an &nbsp to ensure that it does not decode.